### PR TITLE
Support temporary private fork creation via API

### PR DESCRIPTION
**Pull request by Kiyo510**
resolved: #3007

### CODE REVIEW

File: github/security_advisories.go
Comment (gmlewis) [inline github/security_advisories.go:152]:
```suggestion
// CreateTemporaryPrivateFork creates a temporary private fork to collaborate on fixing a security vulnerability in your repository.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @Kiyo510 !
One minor nit, please, and then we should be ready for a second LGTM+Approval from any other contributor to this repo before merging.

**Comment by Kiyo510** [comment]
@gmlewis 
Thanks for reviewing my PR.
Since I made a modification, please check it out again.

**Review by gmlewis — APPROVED** [review]
Thanks, @Kiyo510 !
LGTM.

Awaiting a second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
